torchprime/metrics/tests/test_mfu.py

```diff
@@ -1,151 +1,68 @@
+import sys
+from pathlib import Path
+
 import pytest
+import yaml
 
 from torchprime.metrics.mfu import compute_mfu
 
+ROOT = Path(__file__).resolve().parents[3]
```

Comment:
This looks terribly hacky, could you check if hydra+omegaconf could be leveraged for this?
As far as I understand hydra has tools to be able to find configs located at right place